store/src/App.js

```diff
@@ -31,11 +31,13 @@ import Products from "./pages/product/Products";
 
 import UpdateUserProfile from "./pages/user/UpdateUserProfile";
 import UserProfile from "./pages/user/UserProfile";
+import ScrollToTop from "./components/ScrollToTop";
 
 function App() {
   return (
     <Router>
       <Navbar />
+      <ScrollToTop />
```

Comment:
`antd` already provides an inbuilt component for this. [https://ant.design/components/back-top/](https://ant.design/components/back-top/)
Why create another one?

Reply:
This is not same , it is required for when we navigate another page then it may shows middle of page or last of those page , but adding this it will show another page top part